<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AMC4030</class>
 <widget class="QMainWindow" name="AMC4030">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="Btn_ConnectController">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>连接控制器</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_About">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>关于</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_ParameterSet">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>参数设置 </string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_ManualOperate">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>手动操作</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_ProgramEdite">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>程序编辑</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>721</width>
      <height>401</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>719</width>
       <height>399</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>当前状态：未连接控制器！</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>0</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>870</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu11">
    <property name="title">
     <string>Language</string>
    </property>
    <addaction name="actionChinese"/>
    <addaction name="separator"/>
    <addaction name="actionEnglish"/>
   </widget>
   <addaction name="menu11"/>
  </widget>
  <action name="actionChinese">
   <property name="text">
    <string>中文</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="text">
    <string>English</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
